rtm/plotting.py

```diff
@@ -421,8 +421,8 @@ def plot_stack_peak(S, plot_max=False, ax=None):
                        color='red', edgecolor='black', s=150, zorder=5)
 
     ax.set_xlim(S.time[0].data, S.time[-1].data)
-    ax.set_xlabel('UTC Time')
-    ax.set_ylabel('Peak Stack Amplitude')
+    ax.set_xlabel('UTC time')
+    ax.set_ylabel('Stack amplitude at spatial stack max')
```

Comment:
This results in a really long label that extends well beyond the axes limit, so I reverted to something shorter. Can add more info in the text or caption.

Reply:
:+1: